src/tests/JIT/opt/Compares/conditionalIncrements.cs

```diff
@@ -75,6 +84,14 @@ static long cinc_long_min_max(long op1)
         return op1 < 49 ? long.MinValue : long.MaxValue;
     }
 
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    static long cinv_long_min_max(long op1)
```

Comment:
```suggestion
    static long cinc_long_min_max(long op1)
```

Reply:
This is another invert example that emits `cinc`.